common/src/lib.rs

```diff
@@ -759,9 +779,10 @@ pub struct Node {
     pub busy: bool,
 
     /// The object functions as a text field which exposes its descendants.
+    ///
     /// Use cases include the root of a content-editable region, an ARIA
     /// textbox which isn't currently editable and which has interactive
-    /// descendants, and a <body> element that has "design-mode" set to "on".
+    /// descendants, and a `<body>` element that has "design-mode" set to "on".
```

Comment:
This is what motivated me to make this PR, as the unescaped `<body>` tab [breaks `docs.rs` rendering](https://docs.rs/accesskit/0.3.0/accesskit/struct.Node.html) (scroll down a bit).